src/background/cryptKeeper.ts

```diff
@@ -29,6 +30,8 @@ export default class CryptKeeperController extends Handler {
 
   private historyService: HistoryService;
 
+  private backupService: BackupService;
+
   constructor() {
     super();
     this.requestManager = new RequestManager();
```

Comment:
@0xmad  Do you think we can Singleton RequestManager as well?

Reply:
It depends. For instance, if we are going to reuse it in another service it makes sense to have separate instances.